packages/store/src/internal/state-factory.ts

```diff
@@ -345,10 +346,14 @@ export class StateFactory implements OnDestroy {
     const { dispatched$ } = this._actions;
     for (const actionType of Object.keys(actions)) {
       const actionHandlers = actions[actionType].map(actionMeta => {
-        const cancelable = !!actionMeta.options.cancelUncompleted;
+        const abortController = new AbortController();
```

Comment:
If cancellation is enabled we would retain a reference to the previous `abortController` for the handler and when the next action hits this handler we would call `abortController.abort()` on the previous `abortController` to trigger the cancellation of the previous handler and of any related work that has used the abort signal from the context.
When the abortSignal fires for some work in progress in the handler (like `fetch`), the handler would throw an error, we should catch this error and not let it propagate to the action error handlers.